public/emails/generic.liquid

```diff
@@ -0,0 +1,7 @@
+---
+from: "{{ data.from }}"
+layout: modules/core/mailer
```

Comment:
How can we change a layout in application?

Reply:
by providing one for the email template partial.
`/emails/generic` and `layouts/mailer` are just generic fallback templates in case the `template` param is not specified in the command. so kind of a failsafe that provides a fallback, but the developer should specify the template parameter (I added that part to the docs)

Reply:
I'm wondering about not allowing to overwrite the template itself, and always use the `generic` here, which would act only as an argument forwarded to the graphql.

I'm thinking about extending the generic template with all the options like `cc`, `bcc`, having them null by default, also having layout false by default (if possible? not sure about that one), and allow to overwrite the path to the layout and also to provide partial. I'm thinking about achieving the following outcome:

```
{% parse_json object %}
  {
    "to":      "grievous@example.com",
    "from":    "kenobi@example.com",
    "cc": [],
    "bcc": [],
    "subject": "Hello there!",
    "layout": "path/to/my_layout",
    "partial": "path/to/email_partial",
    "data": { "user": " { "first_name:" "John" } }
  }
{% endparse_json %}
{% function _ = 'modules/core/lib/commands/email/send', object: object %}
```

There shouldn't be a need to provide different email templates, like for example `app/emails/my_email` , because the generic would look pretty much the same as now:

```
---
from: "{{ object.from }}",
layout: "{{ object.layout }}",
subject: "{{ object.subject }}",
cc: "{{ object.cc}}",
bcc: "{{ object.bcc}}",
to: "{{ object.to }}"
---
{% include object.partial, data: object.data %}
```

Reply:
FYI i'm also adding the build & check steps to the send command

Reply:
@Slashek are you sure the platform supports dynamic layout files? 
```Layout Could not find Layout with layout: {{ object.layout }}```

Reply:
oh! you are right! created PR https://github.com/mdyd-dev/desksnearme/pull/10890 cc @piotrze